src/transformers/models/character_bert/modeling_character_bert.py

```diff
@@ -0,0 +1,1938 @@
+        self.transform = CharacterBertPredictionHeadTransform(config)
+
+        # The output weights are the same as the input embeddings, but there is
+        # an output-only bias for each token.
+        self.decoder = nn.Linear(config.hidden_size, config.mlm_vocab_size, bias=False)
+
+        self.bias = nn.Parameter(torch.zeros(config.mlm_vocab_size))
+
+        # Need a link between the two variables so that the bias is correctly resized with `resize_token_embeddings`
+        self.decoder.bias = self.bias
+
+    def forward(self, hidden_states):
+        hidden_states = self.transform(hidden_states)
+        hidden_states = self.decoder(hidden_states)
+        return hidden_states
+
+
+# Copied from transformers.models.bert.modeling_bert.BertOnlyMLMHead with Bert->CharacterBert
+class CharacterBertOnlyMLMHead(nn.Module):
+    def __init__(self, config):
+        super().__init__()
+        self.predictions = CharacterBertLMPredictionHead(config)
+
+    def forward(self, sequence_output):
+        prediction_scores = self.predictions(sequence_output)
+        return prediction_scores
+
+
+# Copied from transformers.models.bert.modeling_bert.BertOnlyNSPHead with Bert->CharacterBert
+class CharacterBertOnlyNSPHead(nn.Module):
+    def __init__(self, config):
+        super().__init__()
+        self.seq_relationship = nn.Linear(config.hidden_size, 2)
+
+    def forward(self, pooled_output):
+        seq_relationship_score = self.seq_relationship(pooled_output)
+        return seq_relationship_score
+
+
+# Copied from transformers.models.bert.modeling_bert.BertPreTrainingHeads with Bert->CharacterBert
+class CharacterBertPreTrainingHeads(nn.Module):
+    def __init__(self, config):
+        super().__init__()
+        self.predictions = CharacterBertLMPredictionHead(config)
+        self.seq_relationship = nn.Linear(config.hidden_size, 2)
+
+    def forward(self, sequence_output, pooled_output):
+        prediction_scores = self.predictions(sequence_output)
+        seq_relationship_score = self.seq_relationship(pooled_output)
+        return prediction_scores, seq_relationship_score
+
+
+class CharacterBertPreTrainedModel(PreTrainedModel):
+    """
+    An abstract class to handle weights initialization and
+    a simple interface for downloading and loading pretrained models.
+    """
+
+    config_class = CharacterBertConfig
+    load_tf_weights = None
+    base_model_prefix = "character_bert"
+    _keys_to_ignore_on_load_missing = [r"position_ids"]
+
+    def _init_weights(self, module):
+        """ Initialize the weights """
+        if isinstance(module, CharacterCnn):
+            # We need to handle the case of this parameters since it is not an actual module
+            module._char_embedding_weights.data.normal_()
+            module._char_embedding_weights.data[0].fill_(0.)  # token padding
+            module._char_embedding_weights.data[
+                CharacterMapper.padding_character + 1].fill_(0.)  # character padding
```

Comment:
```suggestion
            # character padding
            module._char_embedding_weights.data[CharacterMapper.padding_character + 1].fill_(0.)
```
This is less readable when the comment after move the line.